bin/init.bash

```diff
@@ -82,6 +82,15 @@ generate_base_sc_config() {
     if [[ -f $file ]]; then
         yq merge "$tmpfile" "$file" --inplace -a=overwrite --overwrite --prettyPrint
     fi
+
+    cloud_provider=$(yq r -e "$tmpfile" 'global.cloudProvider')
```

Comment:
This file has `CK8S_CLOUD_PROVIDER` as a requirement, so you could use that variable instead of creating this new one